examples/lang/os.mcl

```diff
@@ -9,19 +9,21 @@ file "${tmpdir}/execinfo" {
 	content => fmt.printf("mgmt is at %s\n", os.executable()),
 }
 
+$home = os.getenv("HOME")
 file "${tmpdir}/mgmtenv" {
 	state => $const.res.file.state.exists,
-	content => os.expand_env("$HOME sweet ${os.getenv(\"HOME\")}\n"),
```

Comment:
Certainly worth a mention in the documentation that this specific behavior isn't allowed (any more)

Reply:
Mentioned in new commit message. (Coming shortly.)